spec/unit/utils/encryptable_move_handler_spec.rb

```diff
@@ -26,7 +26,7 @@
   end
 
   it 'moves credential from team2 folder to team1 folder' do
-    credential = encryptables(:credentials2)
+    credential = encryptables(:credentials3)
```

Comment:
Is this because you moved the fixtures around?

Reply:
We changed it, because a shared_file is attached to credentials2. And because we need to hardcode the encrypted data in fixtures, the MoveHandler couldn't decrypt this file. I created a new credential with a new attached file, this is working and tests the method